build.gradle

```diff
@@ -26,6 +26,10 @@ def useCheckerFramework = rootProject.hasProperty('checkerFramework')
 def useErrorProne = !useCheckerFramework
 
 subprojects {
+    // OC-Agent exporter depends on grpc-core which depends on errorprone annotations, and it
+    // doesn't work with checker framework.
+    def mustUseErrorprone = project.name == "opencensus-exporter-trace-ocagent"
```

Comment:
Why not change only the useErrorProne = !useCheckerFramework || project.name == ...?

Reply:
For extensibility. Any artifacts that depend on grpc-core should use error-prone instead of nullness checker. In the future if we add project foo that depends on grpc-core, we can simply add it to this line:
```gradle
def mustUseErrorprone = project.name == "opencensus-exporter-trace-ocagent" || project.name == "any-artifact-with-grpc-core" || ...
```

Reply:
Sorry ignore the previous comment. You're right there's no need to have another variable `mustUseErrorprone`.